tap_ilevel/singer_operations.py

```diff
@@ -83,7 +87,7 @@ def write_bookmark(state, stream, value):
     if 'bookmarks' not in state:
         state['bookmarks'] = {}
     state['bookmarks'][stream] = value
-    LOGGER.info('Write state for stream: %s, value: %s', stream, value)
+    LOGGER.info('Write state for stream: {}, value: {}'.format(stream, value))
```

Comment:
Using `format` violates pylint for these logger changes. This should stay as it was.

Reply:
Ok - changed back.